src/tests/execute/execute_match_tests.rs

```diff
@@ -2201,26 +2181,39 @@ mod execute_match_tests {
                         10,
                         "quote_1",
                         Addr::unchecked("asker"),
-                        Addr::unchecked(MOCK_CONTRACT_ADDR)
+                        Addr::unchecked(MOCK_CONTRACT_ADDR),
+                        Addr::unchecked(MOCK_CONTRACT_ADDR),
                     )
                     .unwrap()
+                    .try_into()
+                    .unwrap()
                 );
                 assert_eq!(
                     execute_response.messages[1].msg,
-                    CosmosMsg::Bank(BankMsg::Send {
-                        to_address: "bidder".into(),
-                        amount: vec![coin(5, "base_1")],
-                    })
+                    // TODO is this okay? why does 'main' branch assert against BankMsg and not transfer_marker_coins?
+                    transfer_marker_coins(
+                        5,
+                        "base_1",
+                        Addr::unchecked("bidder"),
+                        Addr::unchecked(MOCK_CONTRACT_ADDR),
+                        Addr::unchecked(MOCK_CONTRACT_ADDR),
+                    )
+                    .unwrap()
+                    .try_into()
+                    .unwrap()
```

Comment:
I'm not entirely sure if this change is okay. Is there a reason why the assert here isn't the same as the surrounding asserts on `execute_response.messages[0].msg` and `execute_response.messages[2].msg`?

Making this change resolves this failure: 
```
assertion `left == right` failed
  left: Stargate { type_url: "/provenance.marker.v1.MsgTransferRequest", value: Binary(0a0b0a06626173655f311201351a0f636f736d6f7332636f6e7472616374220f636f736d6f7332636f6e74726163742a06626964646572) }
 right: Bank(Send { to_address: "bidder", amount: [Coin { 5 "base_1" }] })
 ```

Reply:
oooh I see that makes a lot of sense. Yeah it's related to the mock issue then. I'll change back these asserts, comment them out, and add a todo